create-prs-to-update-vcs-repositories/create_prs_to_update_vcs_repositories.py

```diff
@@ -1,3 +1,73 @@
+'''
+Description:
+
+    In this script, we create a PR to update the version of the repository specified by the URL in autoware.repos.
+    The steps are as follows:
+        1. Get the repositories with semantic version tags
+            1-1. Currently we only support the semantic version pattern of r'\b(?<![^\s])\d+\.\d+\.\d+(?![-\w.+])\b'.
+                This pattern matches/mismatches for the following examples:
+
+                    "0.0.1",                # match
+                    "0.1.0",                # match
+                    "1.0.0",                # match
+                    "2.1.1",                # match
+                    "v0.0.1",               # mismatch
+                    "ros2-v0.0.4",          # mismatch
+                    "xxx-1.0.0-yyy",        # mismatch
+                    "v1.2.3-beta",          # mismatch
+                    "v1.0",                 # mismatch
+                    "v2",                   # mismatch
+                    "1.0.0-alpha+001",      # mismatch
+                    "v1.0.0-rc1+build.1",   # mismatch
+                    "2.0.0+build.1848",     # mismatch
+                    "2.0.1-alpha.1227",     # mismatch
+                    "1.0.0-alpha.beta",     # mismatch
+                    "ros_humble-v0.10.2"    # mismatch
+
+        2. Get the latest tag
+        3. If the latest tag is newer than the current version, create a PR to update the version in autoware.repos
+            3-1. We can create PRs for major, minor, or patch updates
+                For example, if you want to create a PR only for major updates, you can use the --major option as follows:
+
+                ```
+                python create_prs_to_update_vcs_repositories.py --major
+                ```
+
+                If you want to create PRs for major and minor updates, you can use both the --major and --minor options as follows:
+
+                ```
+                python create_prs_to_update_vcs_repositories.py --major --minor
+                ```
+
+Example usage:
+
+    Example 1: Create PRs for major updates
+    ```
+    python create_prs_to_update_vcs_repositories.py --autoware_repos_file_name autoware.repos --targets major
+    ```
+
+    Example 2: Create PRs for minor updates
+    ```
+    python create_prs_to_update_vcs_repositories.py --autoware_repos_file_name autoware.repos --targets minor
+    ```
+
+    Example 3: Create PRs for patch updates
+    ```
+    python create_prs_to_update_vcs_repositories.py --autoware_repos_file_name autoware.repos --targets patch
+    ```
+
+    Example 4: Create PRs for major and minor updates
+    ```
+    python create_prs_to_update_vcs_repositories.py --autoware_repos_file_name files.repos/autoware.repos --targets major minor
+    ```
+
+    Example 5: Create PRs for any updates
+    ```
+    python create_prs_to_update_vcs_repositories.py --autoware_repos_file_name autoware.repos --targets any
+    ```
+'''
+
+
```

Comment:
The comment and the README.md content are duplicated so that I think the comment is not necessary.

Reply:
@youtalk Thank you so much for your review! Yes, you are right, the information is duplicated to that of `README.md`. I entirely removed the duplicated description from this script.